.github/workflows/scheduled.yml

```diff
@@ -202,6 +202,7 @@ jobs:
         run: |
           EXTRA_CMAKE_FLAGS="-DPYTHON_EXECUTABLE=$(which python3) $EXTRA_CMAKE_FLAGS"
           make debug
+          make release
```

Comment:
Why? We run all other fuzzers debug only.

Reply:
Release build runs faster and could cover more cases, we probably should run all fuzzers in release build

Reply:
I would be fine with either or but running both seems redundant.
Also should we use release with debug symbols for better stacktraces or do the fuzzers have their own thing?

Reply:
Yeah we should include the full symbol in these builds for fuzzers

Reply:
Hi @assignUser and @Yuhta, I discussed with @kgpai offline and we decided to enable the fuzzer test in release mode internally first. We'll set up the release-mode fuzzer tests on CI later if needed. The reason is that the release build on CI may have a slightly different setup from our internal build (according to @kgpai) which could make fuzzer failures potentially hard to reproduce. So we'll ensure it works internally first.

Reply:
FYI , I agree with posters above that we should enable release mode fuzzers with debug symbols - we get it by default on internal so its easier to go with that for the moment , plus we can run it for far longer without having to worry about our CI budget.